RELEASE.md

```diff
@@ -0,0 +1,246 @@
+* [ghcr.io/deephaven/server-slim](https://github.com/deephaven/deephaven-core/pkgs/container/server-slim)
+
+The extended images are all extensions to the `server` image and include additional Python dependencies.
+* [ghcr.io/deephaven/server-all-ai](https://github.com/deephaven/deephaven-core/pkgs/container/server-all-ai)
+* [ghcr.io/deephaven/server-nltk](https://github.com/deephaven/deephaven-core/pkgs/container/server-nltk)
+* [ghcr.io/deephaven/server-pytorch](https://github.com/deephaven/deephaven-core/pkgs/container/server-pytorch)
+* [ghcr.io/deephaven/server-sklearn](https://github.com/deephaven/deephaven-core/pkgs/container/server-sklearn)
+* [ghcr.io/deephaven/server-tensorflow](https://github.com/deephaven/deephaven-core/pkgs/container/server-tensorflow)
+
+### Deephaven server SBOM
+The Deephaven server software bill-of-materials is an experimental artifact that some consumers may want.
+It may also be useful for release management purposes to ensure that no unexpected dependencies are change from release to release.
+It is released to [GitHub releases](https://github.com/deephaven/deephaven-core/releases) in the [syft](https://github.com/anchore/syft) json format.
+
+### Deephaven python client
+The Deephaven python client is released as the `pydeephaven` wheel at [PyPi](https://pypi.org/project/pydeephaven/).
+
+### Deephaven go client
+The Deephaven go client is released as a [Go package](https://pkg.go.dev/github.com/deephaven/deephaven-core/go).
+
+## Release process
+
+The majority of the release procedure is controlled through the [publish-ci.yml workflow](./.github/workflows/publish-ci.yml).
+It is kicked off by a push to a branch name that matches `release/v*`.
+Please familiarize yourself with the steps in that workflow.
+
+### 0. Poll Deephaven team
+
+Ensure you are proceeding with a known release, and there aren't any blockers.
+
+### 1. Repository prerequisites
+
+These release notes assume that the Deephaven repository `git@github.com:deephaven/deephaven-core.git` is referenced as the remote named `upstream`.
+Please ensure your local repository is setup as such, or that you replace any commands with the appropriately named remote:
+```shell
+$ git remote get-url upstream
+git@github.com:deephaven/deephaven-core.git
+```
+
+### 2. Create a local release branch
+
+Ensure you are up-to-date with `upstream/main`, or at the commit that you want to start a new release from.
+If you are unsure what commit to start from, please ask.
+Please double-check you are on the version you expect to be releasing.
+The releases have so far proceeded with `releave/vX.Y.Z`, where `X.Y.Z` is the version number (this isn't a technical requirement), please replace `X.Y.Z` with the appropriate version.
+We also separate out the release branch from `upstream/main` with an empty commit (this isn't a technical requirement).
+
+```shell
+$ git fetch upstream
+$ git checkout upstream/main
+$ ./gradlew printVersion -q
```

Comment:
This command gives me an error.  Is it just to get Gradle downloaded?  Is there another task that could be run?

Reply:
This command only works on this branch - it's something new I've added just as a little helper for release management purposes.